pkg/model/components/etcdmanager/model.go

```diff
@@ -201,6 +200,18 @@ spec:
       name: bin
   hostNetwork: true
   hostPID: true # helps with mounting volumes from inside a container
+  initContainers:
+  - args:
+    - /ko-app/kops-copy
+    - /opt/bin
+    command:
+    - /ko-app/kops-copy
+    image: registry.k8s.io/kops/kops-copy:1.27.0-alpha.2
+    name: kops-copy
+    resources: {}
+    volumeMounts:
+    - mountPath: /opt
+      name: bin
```

Comment:
Maybe this should be `kops-copy` to avoid conflicts (in case an image has an /opt/bin) & to "leave room" for copying other images?

Reply:
We use this volume for all binaries, including etcd ones. Probably naming it `opt` would be better.